engine/src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -188,12 +189,14 @@ class WorkflowManagerActor(config: Config)
   }
 
   private def pushToMetadataService(workflowId: WorkflowId): Unit = {
-    val curTime = currentTime
+    val curTime = OffsetDateTime.now
     val metadataEventMsgs = List(
       MetadataEvent(MetadataKey(workflowId, None, WorkflowMetadataKeys.Id), MetadataValue(workflowId.toString), curTime),
-      MetadataEvent(MetadataKey(workflowId, None, WorkflowMetadataKeys.SubmissionTime), MetadataValue(curTime.asJodaString), curTime),
+      MetadataEvent(MetadataKey(workflowId, None, WorkflowMetadataKeys.SubmissionTime),
+        MetadataValue(curTime.toString), curTime),
       // Currently, submission time is the same as start time
-      MetadataEvent(MetadataKey(workflowId, None, WorkflowMetadataKeys.StartTime), MetadataValue(curTime.asJodaString), curTime)
+      MetadataEvent(MetadataKey(workflowId, None, WorkflowMetadataKeys.StartTime), MetadataValue(curTime.toString),
```

Comment:
TL;DR Yes, they are functionally equivalent, and will print with time offset "Z" on systems running on UTC time.

I'll admit, tests are severely lacking. For now (recalling that `curTime` was a `Timestamp`, and now is a `OffsetDateTime`):

```scala
scala> import java.sql._
import java.sql._

scala> import java.time._
import java.time._

scala> import java.time.format._
import java.time.format._

scala> val instant = Instant.now
instant: java.time.Instant = 2016-05-31T14:43:17.716Z

scala> val timestamp = Timestamp.from(instant)
timestamp: java.sql.Timestamp = 2016-05-31 10:43:17.716

scala> val jodaString = timestamp.toLocalDateTime.atZone(ZoneId.systemDefault).format(DateTimeFormatter.ISO_OFFSET_DATE_TIME)
jodaString: String = 2016-05-31T10:43:17.716-04:00

scala> val offsetDateTime = timestamp.toLocalDateTime.atZone(ZoneId.systemDefault).toOffsetDateTime
offsetDateTime: java.time.OffsetDateTime = 2016-05-31T10:43:17.716-04:00

scala> offsetDateTime.withOffsetSameInstant(ZoneOffset.UTC)
res2: java.time.OffsetDateTime = 2016-05-31T14:43:17.716Z

scala> 
```